active/NavigationView/BottomItems/BottomMenuItems2.md

```diff
@@ -0,0 +1,194 @@
+
+# Background
+Today, [NavigationView](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/navigationview)
+provides a place in its
+[Pane](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/navigationview) 
+to show any number of items.
+The primary purpose of these items is to be a list of navigation links within the app that hosts NavigationView,
+likely as a root element.
+This navigation list is top aligned when NavigationView is shown in one of its left 
+[display modes](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/navigationview#display-modes), 
+and left aligned when NavigationView is in its top 
+[display mode](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/navigationview#display-modes).
+See the left-aligned NavigationView example below for reference:
+
+![NavigationView with labels](./leftnav-anatomy2.png)
```

Comment:
Is Settings really _inside_ the PaneFooter?

Reply:
@MikeHillberg 

I'm waiting on design to provide new "anatomy" visuals for NavigationView docs. Otherwise, this spec is ready to merge.